ntoskrnl/mm/balance.c

```diff
@@ -272,7 +272,8 @@ MmRebalanceMemoryConsumers(VOID)
 {
     // if (InterlockedCompareExchange(&PageOutThreadActive, 0, 1) == 0)
     {
-        KeSetEvent(&MiBalancerEvent, IO_NO_INCREMENT, FALSE);
+		/* Changed increment from IO_NO_INCREMENT to EVENT_INCREMENT to boost balancer priority */
```

Comment:
regarding that comment: Do always indent with spaces, and never with tabs. This must be fixed before merging the PR.

Reply:
Switched it to spaces